.github/workflows/php-cs-fixer.yml

```diff
@@ -0,0 +1,28 @@
+name: Check and fix code styling with php-cs-fixer
+
+on:
+  pull_request:
+  push:
```

Comment:
This doesn't makes sense. If any filter it should do on every push but only on PR to the main branches. But I wouldn't filter at all.

Reply:
on: [ pull_request, push ]


@Gummibeer  is this fine ?